.idea/AlgorytmyNumeryczne.iml

```diff
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.6 (venv)" jdkType="Python SDK" />
```

Comment:
Kurde te pliki trzeba dodac do .gitignore, bo sązę, że jak to zaakceptuje, to mi projekt nie będzie działać XD

Reply:
uh możliwe